### DISCUSSION TITLE
Scoop Logo

### DISCUSSION THREAD

**Discussion by xmha97** [ScoopInstaller/Scoop#5516] [Show and tell]
I need an SVG version of the Scoop logo.
I created it but it is not like original logo.

![z](https://github.com/ScoopInstaller/Scoop/assets/23485114/4d5993dd-69f7-4b0e-a2b1-8c6b6dd9e04c)

**Comment by rashil2000** [comment]
Hey, how did you create this one?

**Reply by xmha97** [reply]
I am not a designer, but I tried to make this logo with [Inkscape](https://inkscape.org/) app.

**Comment by NoPlagiarism** [comment]
Thanks for doing this. I've kinda optimized it by removing styles and replacing them with `fill`

https://github.com/ScoopInstaller/Scoop/assets/37241775/111b97ba-258b-4350-81bb-628ee781337f

**Comment by sitiom** [comment]
See https://github.com/ScoopInstaller/Scoop/issues/2261#issuecomment-3692553470

>I've recreated the Scoop logo in SVG using the original vector source (thanks, free trial) with Scoop's color palette:
>
>| **Logo**<br>https://scoop.sh/logo/scoop-bg.svg            | **Logo (No Background)**<br>https://scoop.sh/logo/scoop.svg                                                                                     |
>|------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------|
>| <img src="https://github.com/user-attachments/assets/994a8dd1-0bb4-42de-a926-607d773726a6" alt="Scoop Logo" width="350"> | <img src="https://github.com/user-attachments/assets/64fa36aa-03f9-481f-a8d4-4f66fe4558ca" alt="Scoop Logo - No Background" width="350"> |
